### DISCUSSION TITLE
Help with starting working with kubric VSCODE Windows

### DISCUSSION THREAD

**Discussion by barakooda** [google-research/kubric#187] [Q&A] [answered]
Hi there,
Im open the project in VSCODE (Windows),
As i understand, it runs the container automatically.

How do I continue from here ?
I`m try running hello world example :

`root@368f1387a6c1:/workspaces/kubric/examples# python3 helloworld.py`
```
Traceback (most recent call last):
  File "/workspaces/kubric/examples/helloworld.py", line 16, in <module>
    import kubric as kb
ModuleNotFoundError: No module named 'kubric'
root@368f1387a6c1:/workspaces/kubric/examples# 
```
  Update:
  As it seem,kubric module not exist so 
  my friend suggest this workaround for now :
  
Edit devcontainer.json
```
  Comment out oull of the image
  Uncomment the build of the image
```
Edit Kubruntu.Dockerfile
```
  Change the Kubric installation part to:
  ADD . /kubric
  RUN pip install .
```

**Comment by MrXandbadas** [comment, marked as answer]
Hey there,
try using WSL2. Make sure your terminal in VSCode is running Ubuntu. The rest of the error codes should help :)
